config-solaris.md

```diff
@@ -26,7 +26,7 @@ The property should consist of a comma-separated privilege set specification as
 ## <a name="configSolarisMaxShmMemory" />maxShmMemory
 The maximum amount of shared memory allowed for this application container.
 A scale (K, M, G, T) can be applied to the value for each of these numbers (for example, 1M is one megabyte).
-Mapped to `max-shm-memory` in [zonecfg(1M)][zonecfg.1m] man page.
+Mapped to `max-shm-memory` in [zonecfg(1M)][zonecfg.1m_2] man page.
```

Comment:
> And I think it makes more sense to use alias `zonecfg.1m` for this link.

You need to use `zonecfg.1m_2` to follow [our policy][1] and avoid conflicts with [the `config.md` entry][2].

[1]: https://github.com/opencontainers/runtime-spec/blame/101e6f9d5c32579c245d8f45f4bdcd87f9a40cf9/style.md#L35
[2]: https://github.com/opencontainers/runtime-spec/blame/101e6f9d5c32579c245d8f45f4bdcd87f9a40cf9/config.md#L838